modules/openshift-developer-cli-pushing-the-init-image-to-a-mirror-registry-on-windows.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * openshift_cli/openshift_developer_cli/using_odo_in_a_restricted_environment/pushing-the-odo-init-image-to-a-mirror-registry.adoc
+
+[id="openshift-developer-cli-pushing-the-init-image-to-a-mirror-registry-on-windows_{context}"]
+
+= Pushing the `init` image to a mirror registry on Windows
+
+.Procedure
+
+. Use `base64` to encode your mirror registry's root certification authority (CA) content:
++
+----
+PS C:\> echo <content_of_additional_ca> | base64 -d > disconnect-ca.crt
+----
+
+. Copy the encoded root CA certificate to the appropriate location. Run PowerShell as administrator and run the following command:
```

Comment:
As an administrator, copy...location by executing the following command in PowerShell: